# Generated by Django 3.2.5 on 2022-01-19 01:53

from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('tasks', '0004_taskhistory_error'),
    ]

    operations = [
        migrations.AddField(
            model_name='taskhistory',
            name='notified_at',
            field=models.DateTimeField(blank=True, null=True),
        ),
        migrations.AddField(
            model_name='taskhistory',
            name='opened_at',
            field=models.DateTimeField(blank=True, null=True),
        ),
        migrations.AddField(
            model_name='taskhistory',
            name='updated_at',
            field=models.DateTimeField(auto_now=True),
        ),
    ]
